icc-x-api/icc-message-x-api.ts

```diff
@@ -1075,6 +1075,10 @@ export class IccMessageXApi extends iccMessageApi {
                 "message",
                 err.toString ? err.toString() : "Server error"
               )
+              const blockingErrors = ["Gateway Timeout", "Failed to fetch"]
```

Comment:
Please specify the scenario leading to these errors and why we think batches were not transmitted to OAs.
Thanks :-)

Reply:
* "Failed to fetch" is due to internet connection lost (shutdown wifi just before sending batch)
* "Gateway Timeout" is add to the blocking error list based on the user feedback (including Frederic)